deployment/helm/node-feature-discovery/values.yaml

```
@@ -13,6 +13,8 @@ namespaceOverride: ""
 enableNodeFeatureApi: false
 
 master:
+  # The TCP port that nfd-master listens for incoming requests. Default: 8080
+  port: 8000
```

We probably don't want to change the default
```suggestion
  port: 8090
```